### DISCUSSION TITLE
The clangd plugin of the code server used a lot of memory, is there any way to optimize it?

### DISCUSSION THREAD

**Discussion by liyongshun** [clangd/clangd#1612] [Q&A]
The clangd plugin of the code server used a lot of memory, is there any way to optimize it?
Every time a webpage is opened, it takes about 2G of memory, but the code directory corresponding to each page is the same

**Comment by HighCommander4** [comment]
It's hard to answer this question in the abstract. If you provide an example project, and indicate what sort of memory usage you are observing, that would allow reasoning about whether the observed memory usage is expected / how it could be reduced.
